Makefile

```diff
@@ -302,5 +302,5 @@ gowork: ## Generate go.work file to support our multi module repository
 
 .PHONY: operator-lint
 operator-lint: gowork ## Runs operator-lint
-	GOBIN=$(LOCALBIN) go install github.com/gibizer/operator-lint@v0.1.0
+	GOBIN=$(LOCALBIN) go install github.com/gibizer/operator-lint@latest
```

Comment:
from past discussion i recall intentions were to use pinned version of operator-lint so projects can move to more recent version whenver it suits to project requirement. or the decision changed or i missed something?

can you also share the Error which this change is fixing to understand better.

Reply:
Well fixing some erros with pre-commit.. This is not relevant patch to the errors. Closing.